### DISCUSSION TITLE
智慧物流平台-Pickupp🚚

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#96] [General]
**新聞大綱：**
2016年於香港成立的**智慧物流平台Pickupp**，今年7月正式登台，年底前將拓展至對物流有重度需求的雙北地區，以彈性、靈活的營運模式，開創物流3.0時代。

Pickupp瞄準市場供需失衡缺口下所隱藏的商機，透過以**「AI模型」預測及調度訂單**，包括智慧綑綁順路單提升送達效率，依據離峰、尖峰智慧定價等方式，為商家大幅降低物流成本。
此外，Pickupp也積極與物流業者在雲端串接服務方面進行合作，透過平台並搭配Pickupp自由接單配送夥伴，幫助第三方物流業者提升運送效率。

透過「店面共享計畫」，將閒置倉庫和市區零售商家的閒置空間，擔任物流中心或轉運站的角色，再透過大數據運算技術，成功改善物流塞車問題，以「共享形式」創造靈活的營運模型。

Pickupp在台灣已成為PChome的物流及電商合作夥伴，提升電子商務及物流配送的能力。未來也將針對中小型企業甚至個人賣家進行服務，滿足不同類型商品與消費者的物流需求。

**新聞心得：**
能以模型預測搭配上雲端串接的服務，來協助物流運輸真的是一個很好的運用，因為各時段的交通車流量其實不一定，但是能透過計算後選擇出最適合的時機點可以很大的改善效率問題，更快速的抵達運送地點，而且店面共享的計畫感覺又更新奇了，現在已經跟PCHome合作了，那之後應該會有更多公司想一起合作吧！

**新聞來源：**
https://udn.com/news/story/7270/5856899

**Comment by ccathyy** [comment]
在我們的生活中，AI真的無所不在耶!希望智慧物流平台Pickupp之後也能跟蝦皮合作~
